kedro/contrib/io/gcs/parquet_gcs.py

```diff
@@ -156,4 +156,6 @@ def _exists(self) -> bool:
 
     def invalidate_cache(self) -> None:
         """Invalidate underlying filesystem caches."""
-        self._gcs.invalidate_cache(self._filepath)
+        # gcsfs expects a string as filepath. With a PosixPath it will crash.
+        str_filepath = str(self._filepath)
+        self._gcs.invalidate_cache(str_filepath)
```

Comment:
Simply stringifying will suffice.
```suggestion
        self._gcs.invalidate_cache(str(self._filepath))
```